exercises/practice/high-scores/.meta/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "authors": [],
+  "contributors": [
+    "kytrinyx"
```

Comment:
Could you please add my name there as well, since I created the solution?

Reply:
Yes, as with the other exercise I will add you as `author`.